### PR TITLE
Added 0.8.3 Release Notes

### PR DESCRIPTION
**Pull request by MadelineMurray**
## PR description
Added 0.8.3 release notes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,97 @@
...
+- Discovery wiring for `--node-whitelist` [\#365](https://github.com/PegaSysEng/pantheon/pull/365) 
+- Plumb in three more metrics [\#344](https://github.com/PegaSysEng/pantheon/pull/344) 
+- `ProposerSelection` to support multiple IBFT implementations [\#307](https://github.com/PegaSysEng/pantheon/pull/307) 
+- Configuration to support IBFT original and revised [\#306](https://github.com/PegaSysEng/pantheon/pull/306) 
+- Added host whitelist for JSON-RPC. [**Breaking Change**](#breaking-change-to-json-rpc-api) [\#295](https://github.com/PegaSysEng/pantheon/pull/295) 
+- Reduce `Block creation processed cancelled` log message to debug [\#294](https://github.com/PegaSysEng/pantheon/pull/294) 
+- Implement iterative peer search [\#268](https://github.com/PegaSysEng/pantheon/pull/268) 
+- Added RLP enc/dec for PrePrepare, Commit and NewRound messages [\#200](https://github.com/PegaSysEng/pantheon/pull/200) 
+- IBFT block mining [\#169](https://github.com/PegaSysEng/pantheon/pull/169) 
+- Added `--goerli` CLI option [\#370](https://github.com/PegaSysEng/pantheon/pull/370) (Thanks to [Nashatyrev](https://github.com/Nashatyrev))
```
Comment (macfarla) [inline CHANGELOG.md]: To reference a github user put `@` before their name

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,97 @@
...
+- Re-enable clique AT [\#373](https://github.com/PegaSysEng/pantheon/pull/373) 
+- Ignoring acceptance test [\#372](https://github.com/PegaSysEng/pantheon/pull/372) 
+- Changes to support Gradle 5.0 [\#371](https://github.com/PegaSysEng/pantheon/pull/371) 
+- Clique: Prevent out of turn blocks interrupt in-turn mining [\#364](https://github.com/PegaSysEng/pantheon/pull/364) 
+- Time all tasks [\#361](https://github.com/PegaSysEng/pantheon/pull/361) 
+- Rework `VoteTallyCache` to better represent purpose [\#360](https://github.com/PegaSysEng/pantheon/pull/360) 
+- Add an `UNKNOWN` `DisconnectReason` [\#359](https://github.com/PegaSysEng/pantheon/pull/359) 
+- New round validation [\#353](https://github.com/PegaSysEng/pantheon/pull/353) 
+- Update get validators for block hash test to start from block 1 [\#352](https://github.com/PegaSysEng/pantheon/pull/352) 
+- Ignore clique AT for now [\#351](https://github.com/PegaSysEng/pantheon/pull/351)
```
Comment (macfarla) [inline CHANGELOG.md]: these got re-enabled later on - could prob remove this one?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,97 @@
...
+
+### Breaking Change to JSON RPC-API 
+
+From v0.8.3, incoming HTTP requests are only accepted from hostnames specified using the `--host-whitelist` command-line option. If not specified, the default value for `--host-whitelist` is `localhost`. 
+
+If using the URL `http://127.0.0.1` to make JSON-RPC calls, use `--host-whitelist` to specify the hostname `127.0.0.1` or update the hostname to `localhost`. 
+
+If your application publishes RPC ports, specify the hostnames when starting Pantheon. For example:  
+```json
+pantheon --host-whitelist=example.com
```
Comment (macfarla) [inline CHANGELOG.md]: is this `json` meant to be here?

File: CHANGELOG.md (hunk 1 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 3 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: Yep - it's md formatting

**Pull request merged**